app/uk/gov/hmrc/helptosavefrontend/views/time_out.scala.html

```diff
@@ -19,21 +19,21 @@
 @import play.twirl.api.Html
 
 @import uk.gov.hmrc.helptosavefrontend.models.HtsContext
-@import uk.gov.hmrc.helptosavefrontend.controllers.routes
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavefrontend.views.ViewHelpers
 
-@this(main: uk.gov.hmrc.helptosavefrontend.views.html.main_template)
+@this(layout: uk.gov.hmrc.helptosavefrontend.views.html.layout, viewHelpers: ViewHelpers)
 
```

Comment:
ViewHelpers is redundant here and can be removed.

Reply:
thanks - I have updated the template